resources/lib/ApiResponse.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require_once __DIR__ . '/ApiResult.php';
+require_once __DIR__ . '/Arrayable.php';
+
+use FINDOLOGIC\Api\Responses\Json10\Json10Response;
+use FINDOLOGIC\Api\Responses\Json10\Properties\Result;
+use FINDOLOGIC\Api\Responses\Json10\Properties\Request;
+
+class ApiResponse extends Json10Response implements Arrayable
+{
+    /** @var Request */
+    public $request;
+
+    /** @var Result */
+    public $result;
```

Comment:
Can we use typehints in the lib classes?
```suggestion
    public Request $request;

    public Result $result;
```